### ISSUE TITLE
How to set the model.predict use only one core?

### ISSUE THREAD

**Issue by yananchen1989** [keras-team/keras#11042]
I want to use my model to predict 10000 rows every time on CPU. I tried this code below to let the prediction only use one CPU every prediction. But I find that in htop, there are 40 cores working. 
There are overall 80 cores on my machine.
So how to set the config to limit the cores used when predicting ? 
```
session_conf = tf.ConfigProto(
      intra_op_parallelism_threads=1,
      inter_op_parallelism_threads=1)

sess = tf.Session(config=session_conf)

while 1:
    pred = model.predict(validation_x, verbose=1, steps=1)
```

My final intention is to use all the 80 cores to predict to speed up the prediction. At this moment, predicting 10000 rows need 375ms. Any good method to speed up to limit the prediction time within 100 ms ? Thanks.

**Comment by heisenbug601** [comment]
@yananchen1989 

`        import numpy as np
	from numpy import linalg as LA
	from keras.applications.vgg16 import VGG16
	from keras.preprocessing import image
	from keras.applications.vgg16 import preprocess_input
	from keras import backend as K
	
	conf = K.tf.ConfigProto(device_count={'CPU': 2},
	                        intra_op_parallelism_threads=2,
	                        inter_op_parallelism_threads=2)
	K.set_session(K.tf.Session(config=conf))
	
	
	class VGGNet:
	    def __init__(self):
	        # weights: 'imagenet'
	        # pooling: 'max' or 'avg'
	        # input_shape: (width, height, 3), width and height should >= 48
	        self.input_shape = (224, 224, 3)
	        self.weight = 'imagenet'
	        self.pooling = 'max'
	        self.model = VGG16(weights=self.weight, input_shape=(self.input_shape[0], self.input_shape[1], self.input_shape[2]),
	                           pooling=self.pooling, include_top=False)
	        self.model.predict(np.zeros((1, 224, 224, 3)))
	
	    '''
	    Use vgg16 model to extract features
	    Output normalized feature vector
	    '''
	    def extract_feat(self, img_path):
	        img = image.load_img(img_path, target_size=(self.input_shape[0], self.input_shape[1]))
	        img = image.img_to_array(img)
	        img = np.expand_dims(img, axis=0)
	        img = preprocess_input(img)
	        feat = self.model.predict(img)
	        norm_feat = feat[0]/LA.norm(feat[0])
	        return norm_feat
	        # return feat[0]`

maybe you can try like this
